internal/generate/olm-catalog/csv.go

```diff
@@ -62,7 +62,8 @@ type BundleGenerator struct {
 	// makeManifests directs the generator to use 'manifests' as the bundle
 	// dir name.
 	MakeManifests bool
-
+	// InteractivePrompts directs the user to provide inputs for CSV fields
+	InteractivePrompts bool
```

Comment:
For bool vars and methods the command nomenclature is to use has or is in the prefix which make very intuitive that it is a bool. E.g hasInteractivePrompts. Also, what it is checking actually? Is if has some dir? I mean when it is true is because has InteractivePrompts directs? If yes, would not be more intuitive a name such as `hasInteractivePromptsDirs`

Reply:
`InteractivePrompts` reflects the `--interactive` flag which can be provided in the `generate-csv` command. Ill modify the naming to be `hasinteractiveprompt`